android/app/src/main/java/poke/rogue/helper/presentation/type/TypeActivity.kt

```diff
@@ -0,0 +1,31 @@
+package poke.rogue.helper.presentation.type
+
+import android.os.Bundle
+import poke.rogue.helper.R
+import poke.rogue.helper.data.mapper.toResultUiModel
+import poke.rogue.helper.databinding.ActivityTypeBinding
+import poke.rogue.helper.local.dao.TypeDao
+import poke.rogue.helper.presentation.base.BindingActivity
+import poke.rogue.helper.presentation.type.model.TypeMatchedResultUiModel
+import poke.rogue.helper.presentation.type.typeselection.TypeSelectionBottomSheetFragment
+
+class TypeActivity : BindingActivity<ActivityTypeBinding>(R.layout.activity_type) {
+    private val bottomSheet by lazy {
+        TypeSelectionBottomSheetFragment()
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding.ivTypeMyTypeContent.setOnClickListener {
+            displayBottomSheet()
+        }
+
+        val dummy =
+            TypeMatchedResultUiModel("페어리", R.drawable.img_property_tmp_2, true, "강한 타입", TypeDao.allTypes.map { it.toResultUiModel() })
+        binding.includedTypeStrong.bind(dummy)
+    }
+
+    private fun displayBottomSheet() {
+        bottomSheet.show(supportFragmentManager, TypeSelectionBottomSheetFragment.TAG)
+    }
```

Comment:
```suggestion   TypeSelectionBottomSheetFragment().show(supportFragmentManager, TypeSelectionBottomSheetFragment.TAG)
    }
```
위에 제가 남긴 코멘트와 동일합니다.  
 혹은 tag를 통해 해당 프래그먼트가 존재하는지 확인하고 생성하는 것이 바람직할 것입니다.

버튼을 빠르게 클릭하면 앱이 crash 납니다.

https://github.com/user-attachments/assets/897d045c-04f4-4c8c-a0d7-14483ba93a4f


![image](https://github.com/user-attachments/assets/5243e3fb-6f5c-4207-9291-de3d2bf1e86a)